starter-files/starter.html

```diff
@@ -40,17 +40,17 @@ <h1 class="team-sidebar__team-name font-semibold text-xl leading-tight mb-1 trun
       </h1>
 
       <div class="team-sidebar__current-user-indicator flex items-center mb-6">
-        <svg class="h-2 w-2 fill-current text-green mr-2" viewBox="0 0 20 20">
+        <svg aria-hidden="true" class="h-2 w-2 fill-current text-green mr-2" viewBox="0 0 20 20">
           <circle cx="10" cy="10" r="10"></circle>
         </svg>
-        <span class="team-sidebar__current-user-name text-white opacity-50 text-sm">
+        <span class="team-sidebar__current-user-name text-white text-sm">
           Mike North
         </span>
       </div>
     </div>
 
     <div>
-      <svg class="h-6 w-6 fill-current text-white opacity-25" viewBox="0 0 20 20">
+      <svg aria-hidden="true" class="h-6 w-6 fill-current text-white opacity-25" viewBox="0 0 20 20">
```

Comment:
This svg is for the notification button.
![image](https://user-images.githubusercontent.com/4473214/61568297-8e5ea000-aa37-11e9-8d0c-b3273787f35d.png)

And it seems to be inaccessible using keyboard, since it is wrapped by a div instead of something like a button, which is focusable via keyboard.

May be replace the `div` with a `button` ?

Reply:
I'll apply a pattern @MelSumner and I agreed on today, where a `<button disabled>` can be used for things like this. Good a11y catch @shankarsridhar!